src/gp/src/GPMSA.C

```diff
@@ -904,7 +907,9 @@ GPMSAFactory<V, M>::setUpEmulator()
               // No fancy perturbation here
               unsigned int j = ex*numOutputs+outj;
 
-              Wy(i,j) = W_i(outi,outj);
+              Wy(i,j) = W_i(outi,outj) *
```

Comment:
Perhaps this is accounting for the need to scale the user's observation covariance?